cli/flox/src/commands/search.rs

```diff
@@ -49,8 +50,11 @@ pub struct Search {
 // which is TODO.
 // Luckily most flakes don't.
 impl Search {
-    #[instrument(name = "search", fields(json = self.json, show_all = self.all, search_term = self.search_term), skip_all)]
+    #[instrument(name = "search", skip_all)]
     pub async fn handle(self, config: Config, flox: Flox) -> Result<()> {
+        set_tag("json", self.json);
```

Comment:
is the tag always set on the root trace/scope, or just the active one?

Reply:
Always the root, which isn't what I expected but it'll do. Hence the comment on the docstring.

You can see an example [here](https://flox-dev.sentry.io/performance/trace/f3d1361da26005d473d93898a243ba1f/?node=txn-523f69522e9d42908b27e452c5e50085&pageEnd&pageStart&query=project%3Acli-flox+user.id%3Ac3318f2c-9e7a-4377-ac81-161ae892fe79&source=traces&statsPeriod=1h&timestamp=1724401477.964&utc=true) which was generated from this branch with this additional change:
```diff
diff --git a/cli/flox/src/commands/list.rs b/cli/flox/src/commands/list.rs
index 97abe603..92aec75a 100644
--- a/cli/flox/src/commands/list.rs
+++ b/cli/flox/src/commands/list.rs
@@ -268,7 +268,9 @@ impl List {
     ///
     /// Check the implementation docs of [Environment::lockfile] for more
     /// information.
+    #[instrument(skip_all)]
     fn get_lockfile(flox: &Flox, env: &mut dyn Environment) -> Result<LockedManifest> {
+        set_tag("foo", "bar");
         let lockfile = Dialog {
                 message: "No lockfile found for environment, building...",
                 help_message: None,
```